orderer/consensus/etcdraft/membership.go

```diff
@@ -105,6 +114,102 @@ func ComputeMembershipChanges(oldMetadata *etcdraft.BlockMetadata, oldConsenters
 	return result, nil
 }
 
+func validateConsenterTLSCerts(c *etcdraft.Consenter, ordererConfig channelconfig.Orderer) error {
+	clientCert, err := parseCertificateFromBytes(c.ClientTlsCert)
+	if err != nil {
+		return errors.Wrap(err, "parsing tls client cert")
+	}
+	serverCert, err := parseCertificateFromBytes(c.ServerTlsCert)
+	if err != nil {
+		return errors.Wrap(err, "parsing tls server cert")
+	}
+
+	for _, org := range ordererConfig.Organizations() {
+		opts, err := createX509VerifyOptions(org.MSP())
+		if err != nil {
+			return errors.WithMessage(err, "creating x509 verify options")
+		}
+
+		// following the lead of msp/mspimplvalidate.go:
```

Comment:
This is for TLS, so we actually do care about the time. If the CA are expired or too early, the TLS handshake won't work.

Reply:
Ah, I see. Makes sense. I'll fix that. Thanks!

Reply:
Done.